lib/tasks/data.rake

```diff
@@ -0,0 +1,35 @@
+namespace :data do
+  desc 'Run all migrations'
+  task migrations: :environment do
+    Rake.application.in_namespace(:data) do |namespace|
+      namespace.tasks.each do |task|
+        next if task.name == 'data:migrations'
+
+        puts "Invoking #{task.name}"
+        task.invoke
+      end
+    end
+  end
+
+  desc 'Migrate user answers to responses'
+  task migrate_user_answers_to_responses: :environment do
+    puts 'Truncate responses'
+    ActiveRecord::Base.connection.execute('TRUNCATE responses RESTART IDENTITY CASCADE')
+    puts 'Migrating user answers to responses'
+    UserAnswer.all.each do |user_answer|
+      response = Response.create!(
+        user_id: user_answer.user_id,
+        training_module: user_answer.module,
+        question_name: user_answer.name,
+        answers: user_answer.answer,
+        correct: user_answer.correct,
+        assessments_type: user_answer.assessments_type,
+        schema: user_answer.question.schema,
+        answer_text: user_answer.question.answer.json.to_json,
```

Comment:
We don't need to call `to_json` and I'd favour using `answer_json` or just `answers` as the field name.

Reply:
`answers` already exists and is the answers the user gives, whereas `answer_text` is the text of the answers in the question.  Could change to `answer_options' if that makes more sense?